docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,26 @@
+| .NET Standard              | [1.0] | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5]  | [1.6]  | [2.0] |
+|----------------------------|-------|--------|-------|-------|-------|--------|--------|-------|
+| .NET Core                  | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0    | 1.0    | 2.0   |
+| .NET Framework             | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.2* | vNext* | 4.6.1 |
```

Comment:
Seconded! but the whole purpose of docs is to be **not confusing** and this is not confusing. At least for the time being when people migrates to the new tooling and are aware of the changes.